examples/bcoin-p2pkh.js

```diff
@@ -4,84 +4,86 @@ const bledger = require('../lib/bledger');
-        + 'bcc0fd134852bf73b0a52388ac00e1f505000000001976a914'
-        + 'efd047c8d895b76cbdcd84a798695e76a0fdb3e488ac000000'
-        + '00', 'hex'),
-      index: 1,
-      path: 'm/44\'/0\'/0\'/1/0'
-    }, {
-      tx: Buffer.from(
-          '01000000010578b91737c81322628b211fd9507083f9251a9d'
-        + 'e3882d188866a560f94a6c07000000006b483045022100adbf'
-        + '82d96bb743a74eb3331392caed2fd457e3e0599d511e220e17'
-        + 'bf698575fe02200f4eeb7bdbb797a2f788de1bd5a434d1fd61'
-        + 'b250332fefb22095ea6cdb8a0e9a012102cce90d5a8a0a37c2'
-        + 'df09b5fea38d546b0ba58ce7099ea8127721bea333da6a2aff'
-        + 'ffffff0200e1f505000000001976a9143028112a4d60e84316'
-        + '9fa0113d02a8c9425cdf8e88ac44060d8f000000001976a914'
-        + '160d0d2de728f7129a26a09c5d9d48e000a220e188ac000000'
-        + '00', 'hex'),
-      index: 0,
-      path: 'm/44\'/0\'/0\'/0/1'
-    }, {
-      tx: Buffer.from(
-        '0100000001ae29b8f0086276bb51a4abe8c107cfa44500bbd8'
-        + '24f15a1beb86a0d72b27b1d0000000006a47304402200c3111'
-        + 'b71a92677cc6bce922d3e18ea3a983bb4c398ae2fed1fd1b04'
-        + '6d7019f10220371f900279de78f6da5db06afbaad74fbac6be'
-        + '2b264ec79ffea39459edf6f0a80121024519c790d3add67179'
-        + 'ff9f14e45117e4cbed5924f13ae0c5ca17b856795c1439ffff'
-        + 'ffff0200e1f505000000001976a9143028112a4d60e843169f'
-        + 'a0113d02a8c9425cdf8e88acc4f34ba6000000001976a914fc'
-        + '8b01674ecc8cc2314a27f3c224926af36df11788ac00000000', 'hex'),
-      index: 0,
-      path: 'm/44\'/0\'/0\'/0/1'
-    }
+  const accinfo = [
+    { path: 'm/44\'/0\'/0\'/0/0' },
+    { path: 'm/44\'/0\'/0\'/1/0' },
+    { path: 'm/44\'/0\'/0\'/1/1' }
   ];
 
-  const ledgerInputs = inputs.map(i => new LedgerTXInput(i));
+  // we need address to send tx to ourselves.
+  for (const info of accinfo) {
+    const hd = await ledgerBcoin.getPublicKey(info.path);
+    const ring = KeyRing.fromPublic(hd.publicKey);
+    info.address = ring.getAddress('base58');
+
+    // fund with 2 transactions
+    const {coins, txs} = await fundUtil.fundAddress(info.address, 2);
+    info.coins = coins;
+    info.txs = txs;
+  }
 
-  const mtx = MTX.fromRaw(tx, 'hex');
+  const coins = accinfo.reduce((coins, info) => coins.concat(info.coins), []);
+  const spendAmount = coins.length - 1;
+
+  const mtx = new MTX();
+
+  mtx.addOutput({
+    address: '3Bi9H1hzCHWJoFEjc4xzVzEMywi35dyvsV',
+    value: Amount.fromBTC(spendAmount).toValue()
+  });
+
+  await mtx.fund(coins, { changeAddress: accinfo[0].address });
+
+  let ledgerInputs = [];
+
+  for (const info of accinfo) {
+    ledgerInputs = ledgerInputs.concat(info.txs.map((tx, i) => {
+      return new LedgerTXInput({
+        tx: tx,
+        index: info.coins[i].index,
+        path: info.path
+      });
+    }));
+  }
 
   await ledgerBcoin.signTransaction(mtx, ledgerInputs);
 
   console.log(mtx.toRaw().toString('hex'));
 
+  console.log(mtx.verify());
```

Comment:
Maybe give some context to this so its:

`console.log(`Valid transaction: ${mtx.verify()}`)`